packages/plugin-ext/src/common/plugin-protocol.ts

```diff
@@ -58,7 +60,7 @@ export interface PluginPackage {
 }
 export namespace PluginPackage {
     export function toPluginUrl(pck: PluginPackage, relativePath: string): string {
-        return `hostedPlugin/${getPluginId(pck)}/${encodeURIComponent(relativePath)}`;
+        return URI.file(pck.packagePath + '/' + relativePath).toString();
```

Comment:
could we do here instead `file?${URI.file(pck.packagePath + '/' + relativePath).toString()}` to avoid breaking all clients

Reply:
Actually, i think we should leave as is and change clients to something like:
```ts
new Endpoint({ path: iconUrl }).getRestUrl().toString()
```

It won't break anyone and fix clients in this repo.